client/src/core/client/stream/tabs/Notifications/LiveBellIcon.tsx

```diff
@@ -0,0 +1,64 @@
+import cn from "classnames";
+import { useLocal } from "coral-framework/lib/relay";
+import React, { FunctionComponent, useMemo } from "react";
+import { graphql } from "relay-runtime";
+
+import { LiveBellIconLocal } from "coral-stream/__generated__/LiveBellIconLocal.graphql";
+import CLASSES from "coral-stream/classes";
+import { NotificationBellIcon, SvgIcon } from "coral-ui/components/icons";
+import styles from "./LiveBellIcon.css";
+
+interface Props {
+  userID?: string;
+  size?: "md" | "lg";
+}
+
+export const LiveBellIcon: FunctionComponent<Props> = ({ size = "md" }) => {
+  const [{ notificationCount }] = useLocal<LiveBellIconLocal>(graphql`
+    fragment LiveBellIconLocal on Local {
+      notificationCount
+    }
+  `);
+
+  const count = useMemo(() => {
+    if (!notificationCount) {
+      return "";
```

Comment:
What if we returned `null` when there's no `notificationCount` or it's <= 0? This would simplify the check below, could just do `count && ()` on line 52 then.

Reply:
updated!